src/components/app/variablesV2.css

```diff
@@ -101,6 +101,8 @@ or "warn/action" ineastd of "red/green"
   /*************************
       Other Colors
   *************************/
+  --color-console-background: #08223c;
+  --color-console-text: #9bf436;
```

Comment:
Probably this colors codes will be only use eon that screen but if not maybe you can rename the variables to use the color name so that will be more generic.
You can use a tool like this for get the names
http://chir.ag/projects/name-that-color/#9BF436